region/maridia/outer.json

```diff
@@ -3215,9 +3650,32 @@
                       "shinesparkFrames": 65
                     }}
                   ]
+                },
+                {
+                  "name": "Cross Room Jump - Speed Jump",
+                  "notable": false,
+                  "requires": [
+                    "canCrossRoomJumpIntoWater",
+                    {"adjacentJumpway": {
+                      "jumpwayType": "platformBelow",
+                      "fromNode": 2,
+                      "maxHeight": 10,
+                      "maxLeftPosition": -38.5,
+                      "minRightPosition": -2
+                    }},
+                    "SpeedBooster",
+                    "canTrickyJump"
+                  ],
+                  "note": [
+                    "Run and jump up through the door using SpeedBooster in the room below.",
+                    "Turn left immediately after jumping, to allow you to run longer and to avoid hitting the crab."
```

Comment:
Shooting right after jumping, and turning around, works even better.  Plus you hit the transition sooner.